src/otx/algorithms/classification/adapters/mmcls/configurer.py

```diff
@@ -200,78 +196,20 @@ def configure_input_size(
         logger.info("Input size is changed to {}".format(input_size))
 
 
-CLASS_INC_DATASET = [
-    "OTXClsDataset",
-    "OTXMultilabelClsDataset",
-    "MPAHierarchicalClsDataset",
-    "ClsTVDataset",
-]
-WEIGHT_MIX_CLASSIFIER = ["CustomImageClassifier"]
-
-
-class IncrClassificationConfigurer(ClassificationConfigurer):
+class IncrClassificationConfigurer(IncrConfigurerMixin, ClassificationConfigurer):
     """Patch config to support incremental learning for classification."""
 
     def configure_task(self, cfg):
         """Patch config to support incremental learning."""
         super().configure_task(cfg)
         if "task_adapt" in cfg and self.task_adapt_type == "mpa":
             self.configure_task_adapt_hook(cfg)
-
-    def configure_task_adapt_hook(self, cfg):
-        """Add TaskAdaptHook for sampler."""
-        train_data_cfg = self.get_data_cfg(cfg, "train")
-        if self.training:
-            if train_data_cfg.type not in CLASS_INC_DATASET:
-                logger.warning(f"Class Incremental Learning for {train_data_cfg.type} is not yet supported!")
-            if "new_classes" not in train_data_cfg:
-                logger.warning('"new_classes" should be defined for incremental learning w/ current model.')
-
-            if cfg.model.type in WEIGHT_MIX_CLASSIFIER:
-                cfg.model.task_adapt = ConfigDict(
-                    src_classes=self.org_model_classes,
-                    dst_classes=self.model_classes,
-                )
-            else:
-                logger.warning(f"Weight mixing for {cfg.model.type} is not yet supported!")
-
-            train_data_cfg.classes = self.model_classes
```

Comment:
It seems that these lines of code disappear. Is it ok?

Reply:
They just print warning without any changes, so I just deleted them for simpler codes.